### PR TITLE
NEPT-367: Use proper projects for Composer dependencies (phing stuff)

### PR DESCRIPTION
**Pull request by drupol**
Phing Drush Task and Phing Behat Task are now projects on Drupal and Github respectively.

- https://github.com/drupol/phingdrushtask
- https://github.com/drupol/phingbehattask

Both projects has full tests coverage. It might be great to use those tools inside the platform.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {
   "require": {
+    "php": ">=5.4.0",
```
Comment (ademarco) [inline composer.json]: We should require PHP >= 5.6.0 since we use `use function` construct in our custom Behat contexts.

Comment (Fefaine) [inline composer.json]: Also, our servers are runing 5.6 So i would even fix this to 5.6

Comment (drupol) [inline composer.json]: This is out of scope and part of #1044

**Comment by ademarco** [comment]
We decided with @drupol to include this in NEPT-153. Closing this PR as wont fix.

**Comment by drupol** [comment]
Link #1006
